File: java/folder program/ProdukElektronik.java
public class ProdukElektronik {
    private String kode;
    private String nama;
    private int harga;
    private String deskripsi;

    // Constructor default
    public ProdukElektronik() {}

    // Constructor parameter
    public ProdukElektronik(String kode,String nama, int harga, String deskripsi) {
        this.kode = kode;
        this.nama = nama;
        this.harga = harga;
        this.deskripsi = deskripsi;
    }

    // Getter & Setter
     public String getKode() { return kode; }
    public void setKode(String kode) { this.kode= kode; }

    public String getNama() { return nama; }
    public void setNama(String nama) { this.nama = nama; }

    public int getHarga() { return harga; }
    public void setHarga(int harga) { this.harga = harga; }

    public String getDeskripsi() { return deskripsi; }
    public void setDeskripsi(String deskripsi) { this.deskripsi = deskripsi; }

    public void displayInfo() {
        System.out.println("Kode: " + kode);
        System.out.println("Nama: " + nama);
        System.out.println("Harga: Rp" + harga);
        System.out.println("Deskripsi: " + deskripsi);
    }
}

File: cpp/folder program/ProdukElektronik.cpp
#include <iostream>

using namespace std;

class ProdukElektronik{
    private:

    string kode;
    string nama;
    int harga;
    string deskripsi;

    public:

    ProdukElektronik(){

    }
    void setkode(string kode){
        this -> kode=kode;
    }
    string getkode(){
        return this->kode;
    }

    void setnama(string nama){
        this -> nama=nama;
    }
    string getnama(){
        return this->nama;
    }

    void setharga(int harga){
        this->harga=harga;
    }
    int getharga(){
        return this->harga;
    }

    void setdeskripsi(string deskripsi){
        this->deskripsi=deskripsi;
    }
    string getdeskripsi(){
        return this->deskripsi;
    }
    // ProdukElektronik(string nama,int harga,string deskripsi){
    //     this ->nama=nama;
    //     this->harga=harga;
    //     this->deskripsi=harga;
    // }

};

// getter setter


